meeting/14-12-18.md

```diff
@@ -0,0 +1,13 @@
+### Long Term goal:
+- Take more responsibility and manual test our code. (A friendly reminder)
+
+### Short Term goal: (After Christmas release)
+- Try out our new Cocoapod strategy. (Not commit the Pod folder)
+  Next meeting we will discuss how this is working for us.
+
```

Comment:
Fun fact: the "new Cocoapod strategy" is actually the _old_ Cocoapod strategy. 😛